src/rundeck/README.md

```diff
@@ -0,0 +1,75 @@
+# Rundeck Helm chart
+This chart would install Rundeck in kubernetes cluster. Rundeck is open source software that helps automating routine operational procedures in data center or cloud environments. For more details on Rundeck, please refer [Rundeck](http://rundeck.org).
+Chart supports persistent shared store for storing rundeck project related files. It also supports external DB to store various jobs and their logs.
+## Prerequisite
+Kubernetes 1.9+
+Secret with name **rundeck-ssh** should exist in the k8s cluster which should use private key for remote edge nodes passwordless connection.

```

Comment:
MUST: This is not good.  Helm charts are supposed to work outof the box, without prereqs.

Please auto-gen the secret via a template and create in the k8s cluster.  Ensure that the secret is not clobbered upon helm upgrade.

Reply:
Yep. in New PR. Secret are not mandatory now.